packages/react/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -25,13 +25,18 @@ const Breadcrumb = ({
   });
 
   return (
-    <nav className={className} {...rest}>
-      {children}
+    <nav aria-label={ariaLabel ? ariaLabel : 'Breadcrumb'}>
+      <ol className={className}>{children}</ol>
```

Comment:
Should the `ol` have `customClassName` or should the `nav`? Would have assumed `nav` since it's the outermost element

Reply:
Yea the nav would make more sense should we remove it from the ol? My question is since that targets the list specifically (ie flex)?

Reply:
Yea that makes sense thanks for the explanation I've added it to the outer tag